hudi-spark-datasource/hudi-spark/src/test/scala/org/apache/spark/sql/hudi/TestInsertTable.scala

```diff
@@ -2448,6 +2449,50 @@ class TestInsertTable extends HoodieSparkSqlTestBase {
     })
   }
 
+  test("Test query with Foldable Propagation expression") {
```

Comment:
Make the test name more readable

Reply:
Not sure what to change this too. It deals with a specific spark optimization